common/common-ui/src/main/java/com/duckduckgo/common/ui/store/ThemingSharedPreferences.kt

```diff
@@ -56,16 +57,27 @@ class ThemingSharedPreferences @Inject constructor(
             when (theme) {
                 DuckDuckGoTheme.SYSTEM_DEFAULT -> THEME_SYSTEM_DEFAULT
                 DuckDuckGoTheme.LIGHT -> THEME_LIGHT
-                DuckDuckGoTheme.DARK -> THEME_DARK
+                else -> THEME_DARK
```

Comment:
In theory we should never receive the experiment values here, but having a catch-all else clause might be a source of future bugs. I'd rather us handle the experiment cases explicitly and map them to regular light/dark values.

Reply:
I understand that, but if anything changes in the code that calls this function which would make it suddenly pass the experimental value, it would go unnoticed.